packages/deprecated-component-library-helpers/styles/color.scss

```diff
@@ -30,25 +30,25 @@ $ca-palette-peach: #f3786d;
 $ca-palette-yuzu: #ffce1e;
 
 // Tertiary colours
-$ca-palette-stone: add-tint($ca-palette-ink, 97%);
-$ca-palette-ash: add-tint($ca-palette-ink, 94%);
-$ca-palette-positive: add-tint($ca-palette-seedling, 30%);
+$ca-palette-stone: #f9f9f9; // previously add-tint($ca-palette-ink, 97%);
```

Comment:
Just a thought - I wonder if the next phase of this upgrade is to make all of these variables warnings in stylelint?

Reply:
Yeah, that would be good. Shall we make an issue for this? Were you thinking any hardcoded css color that isn't using a token (it would have to ignore the token files of course!)? or just ones with "previously add-tint" in the comments?